eth/tracers/api_test.go

```diff
@@ -76,6 +75,17 @@ func (m *mockHistoricalBackend) TraceBlockByNumber(ctx context.Context, number r
 	return nil, ethereum.NotFound
 }
 
+func (m *mockHistoricalBackend) TraceTransaction(ctx context.Context, hash common.Hash, config *TraceConfig) (interface{}, error) {
+	fmt.Println("ran mockHistoricalBackend.TraceTransaction")
```

Comment:
We generally try to avoid printing in tests

Reply:
fixed.